dev/io.openliberty.pages.3.1.internal_fat/test-applications/Misc.war/resources/staticImports.jsp

```diff
@@ -23,13 +23,18 @@
     EL Field expression: ${NAME}
     <br/>
     EL Method expression: ${SPECIALTY()}
+    <br>
+    EL Interface Field expression: ${FIELD}
```

Comment:
I completely missed this. Let's add an assert  to verify the `FIELD` var evaluated via scriplet! 

Sorry if you have this assert before (and I asked to remove it)!